Shared/Samples/Create load report/CreateLoadReportView.Model.swift

```diff
@@ -177,7 +155,7 @@ extension CreateLoadReportView {
             
             guard let phasesNetworkAttribute,
                   let initialExpression,
-                  let traceParameters else { return }
+                  let traceParameters else { throw LoadError.cannotGenerateReport }
```

Comment:
I'm not sure this really helps the user. A few reasons, first it doesn't tell them why.  Second, it allows them to hit the `run` button when the `loadUtilityNetwork()` function actually couldn't find everything it was looking for.  It might be better to fail the setup or `loadUtilityNetwork()` if those variables cannot be filled out.